### PR TITLE
avoid use of 'import' in BFAST easyconfigs, just strip of 'a' from version

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@vanzod please review?

It's less generic, but certainly better than having an `import` statement in there imho...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2156.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fc80849d3b6ca8c5bf64f14878eb3b1b for a full test report.

File: easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-goolf-1.4.10.eb [hunk 1]
```diff
@@ -24,11 +24,8 @@ description = """BFAST facilitates the fast and accurate mapping of short reads
 
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 
-import string
-swdir = version.rstrip(string.lowercase + string.uppercase)
-# eg. http://sourceforge.net/projects/bfast/files/bfast/0.7.0/bfast-0.7.0a.tar.gz/download
+source_urls = ['http://sourceforge.net/projects/bfast/files/bfast/%s/' % version.rstrip('a'), 'download']
```
Comment (vanzod) [inline easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-goolf-1.4.10.eb]: An alternative closer to the original which does not involve user intervention:

`swdir = ''.join(char for char in version if not char.isalpha())`
`source_urls = ['http://sourceforge.net/projects/bfast/files/bfast/%s/' % swdir, 'download']`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2156.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/57eb59126fc9d75aabd9d433011f3fb9 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @vanzod!

**Pull request merged**